code/datums/traits/_trait.dm

```diff
@@ -0,0 +1,100 @@
+//every trait in this folder should be coded around being applied on spawn
+//these are NOT "mob traits" like GOTTAGOFAST, but exist as a medium to apply them and other different effects
```

Comment:
So what's a difference between a trait and status effect, because it seems to me you're duplicating the functionality if the primary feature is just processing.

Reply:
Mainly the tiein with mob traits. It might be possible to make them into a status effect subtype, but it'd require significant changes like forcing it to not go into the mob's status effects.

Reply:
Alright